store/extensions/G-EnterFullRoom/extension.json

```
@@ -0,0 +1,44 @@
+{
+    "title": "G-EnterFullRoom",
+    "description": "Enter a full room automatically when someone leave.",
+    "authors": [
+      {
+        "name": "HappyBoyx3",
+        "hotel": ".de",
+        "username": "HappyBoyx3",
+        "discord": "MAIK#5094",
+        "twitter": "HabboMaik"
+      },
+	    
+	{		
+	"name": "JustProDylan",
+      }
+    ],
+    "version": "1.0",
+    "categories": ["Usability", "Others"],
+    
+    "source": "https://github.com/HabboMaik/G-EnterFullRoom",
+    "readme": "https://github.com/HabboMaik/G-EnterFullRoom/blob/main/README.md",
+    "releases": "https://github.com/HabboMaik/G-EnterFullRoom/releases/tag/v1.0"

```

`source`: The repository must contain the source code, not a compiled exe

`releases`: The link gives a 404, it's better to use the overall releases link

```suggestion
    "source": "https://github.com/HabboMaik/G-EnterFullRoom",
    "readme": "https://github.com/HabboMaik/G-EnterFullRoom/blob/main/README.md",
    "releases": "https://github.com/HabboMaik/G-EnterFullRoom/releases"
```